test/getvolume_tester.py

```diff
@@ -127,3 +88,66 @@ def check_response(self, resp):
         # in the flow or not
         mock_3parclient = self.mock_objects['mock_3parclient']
         mock_3parclient.getWsApiVersion.assert_called()
+
+
+class GetSnapshotUnitTest(GetVolumeUnitTest):
```

Comment:
I don't see implementation for this class

Reply:
It's a base class for snapshot-testers like TestSyncSnapshots